packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap

```diff
@@ -0,0 +1,168 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`All Android Dist Files Exist 1`] = `
+Array [
+  "dist/android/classic/color-dark.xml",
+  "dist/android/classic/colors-light.xml",
+  "dist/android/classic/font-size.xml",
+  "dist/android/classic/font-weight.xml",
+  "dist/android/classic/opacity.xml",
+  "dist/android/classic/rounding.xml",
+  "dist/android/classic/space.xml",
+  "dist/android/colors-dark.xml",
+  "dist/android/colors.xml",
+  "dist/android/font-size.xml",
+  "dist/android/opacity.xml",
+  "dist/android/rounding.xml",
+  "dist/android/space.xml",
+  "dist/android/vr/colors-dark.xml",
+  "dist/android/vr/colors.xml",
+  "dist/android/vr/font-size.xml",
+  "dist/android/vr-theme/color-dark.xml",
+  "dist/android/vr-theme/colors-light.xml",
+  "dist/android/vr-theme/font-size.xml",
+  "dist/android/vr-theme/font-weight.xml",
+  "dist/android/vr-theme/opacity.xml",
+  "dist/android/vr-theme/rounding.xml",
+  "dist/android/vr-theme/space.xml",
+]
+`;
+
+exports[`All Web CSS Files Exist 1`] = `
+Array [
+  "dist/css/classic/variables-dark.css",
+  "dist/css/classic/variables.css",
+  "dist/css/variables-dark.css",
+  "dist/css/variables.css",
+  "dist/css/vr/variables-dark.css",
+  "dist/css/vr/variables.css",
+  "dist/css/vr-theme/language/ck/line-height.css",
+  "dist/css/vr-theme/language/en/line-height.css",
+  "dist/css/vr-theme/line-height-ck.css",
+  "dist/css/vr-theme/line-height-en.css",
+  "dist/css/vr-theme/variables-dark.css",
+  "dist/css/vr-theme/variables.css",
+  "dist/css/vr-theme-web-mapping/language/ck/line-height.css",
```

Comment:
this doenst look it should be here heheh

Reply:
oh good catch - I think it's from my local build!